client/core/core.go

```diff
@@ -3178,10 +3209,10 @@ func (c *Core) initialize() {
 	// for authentication when Login is triggered.
 	wg.Wait()
 	c.log.Infof("Successfully connected to %d out of %d DEX servers", len(c.conns), len(accts))
-	for dexName, dc := range c.dexConnections() {
+	for _, dc := range c.dexConnections() {
```

Comment:
Minor bug I spotted - it is now a slice range so the first variable was no longer the dex name string but an integer

Reply:
Resolved separately in https://github.com/decred/dcrdex/pull/977 by @JoeGruffins